scripts/npm/check-owners.js

```diff
@@ -19,23 +19,80 @@
 /* @flow */
 
 /* eslint-disable no-console */
+/* eslint-disable no-use-before-define */
```

Comment:
I do want hoisting in my JS!

Reply:
Unfortunately the rule is there for a reason Matteo :)

Reply:
In this particular instance I want to allow hoisting, it dramatically improve readability having the uniform function on top, opening the file will give a full overview of the business logic, implementation details are further down

Reply:
I'm with @matthewdavidson  on this one. Maybe C# / Java devs would find this more readable.
If the business logic is really much lower-level than the overall business logic, it should probably be split into multiple files.

Reply:
I completely disagree with that, that's not C# / Java at all, splitting the logic in multiple file doesn't really make sense here as won't be re-used by other scripts, at least not for now